src/llmcompressor/modifiers/smoothquant/utils.py

```diff
@@ -28,9 +28,6 @@
         balance_layers=["re:.*q_proj", "re:.*k_proj", "re:.*v_proj"],
         smooth_layers="re:.*input_layernorm",
     ),
-    LayerMap(
```

Comment:
I think it'd be good to leave a TODO and an issue # with the future work you outlined in the message thread. This might be confusing for users.

Reply:
+ a jira ticket